index.html

```diff
@@ -531,6 +545,13 @@ <h3 id="installation-sec">
         <h3>
           <code>BeforeInstallPromptEvent</code> Interface
         </h3>
+        <div class="note">
+          The <code>beforeinstallprompt</code> event is somewhat misnamed, as
+          it does not necessarily signal that an <a>automated install
+          prompt</a> will follow (depending on the user agent, it may just be
```

Comment:
s/it may just/it can just
or "it might just"
(avoid RFC2119 keywords in notes... even in lowercase)